gdsfactory/components/bends/bend_s.py

```diff
@@ -56,8 +57,13 @@ def bezier(
         cross_section: spec.
         bend_radius_error_type: error type.
         allow_min_radius_violation: bool.
+        width: width to use. Defaults to cross_section.width.
     """
-    xs = gf.get_cross_section(cross_section)
+    if width:
+        xs = gf.get_cross_section(cross_section, width=width)
+    else:
+        xs = gf.get_cross_section(cross_section)
+
```

Comment:
i think it's better to set width or cross section, but not both

Reply:
if you try to set a width with a pre-defined cross_section you'll get a warning